src/units/frame.hpp

```diff
@@ -200,7 +200,7 @@ class unit_frame
 {
 public:
 	// Constructors
-	unit_frame(const frame_builder& builder = frame_builder()) : builder_(builder) {}
+	explicit unit_frame(const frame_builder& builder = frame_builder()) : builder_(builder) {}
```

Comment:
The entire purpose of the `frame_builder` class is to produce a `unit_frame` object, so I think this is a case where the implicit conversion is preferred.